Current.vcxproj.filters

```diff
@@ -0,0 +1,1080 @@
+    </ClInclude>
+    <ClInclude Include="TypeSystem\primitive_types.dsl.h">
+      <Filter>Header Files\TypeSystem</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\struct.h">
+      <Filter>Header Files\TypeSystem</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\timestamp.h">
+      <Filter>Header Files\TypeSystem</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\types.h">
+      <Filter>Header Files\TypeSystem</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\variant.h">
+      <Filter>Header Files\TypeSystem</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Reflection\exceptions.h">
+      <Filter>Header Files\TypeSystem\Reflection</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Reflection\reflection.h">
+      <Filter>Header Files\TypeSystem\Reflection</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Reflection\types.h">
+      <Filter>Header Files\TypeSystem\Reflection</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Schema\schema.h">
+      <Filter>Header Files\TypeSystem\Schema</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Schema\smoke_test_struct.h">
+      <Filter>Header Files\TypeSystem\Schema</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Serialization\binary.h">
+      <Filter>Header Files\TypeSystem\Serialization</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Serialization\exceptions.h">
+      <Filter>Header Files\TypeSystem\Serialization</Filter>
+    </ClInclude>
+    <ClInclude Include="TypeSystem\Serialization\json.h">
+      <Filter>Header Files\TypeSystem\Serialization</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\metaprogramming.h">
+      <Filter>Header Files\Yoda</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\sfinae.h">
+      <Filter>Header Files\Yoda</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\types.h">
+      <Filter>Header Files\Yoda</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\yoda.h">
+      <Filter>Header Files\Yoda</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\container\dictionary\api.h">
+      <Filter>Header Files\Yoda\container\dictionary</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\container\dictionary\exceptions.h">
+      <Filter>Header Files\Yoda\container\dictionary</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\container\dictionary\metaprogramming.h">
+      <Filter>Header Files\Yoda\container\dictionary</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\container\matrix\api.h">
+      <Filter>Header Files\Yoda\container\matrix</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\container\matrix\exceptions.h">
+      <Filter>Header Files\Yoda\container\matrix</Filter>
+    </ClInclude>
+    <ClInclude Include="Yoda\container\matrix\metaprogramming.h">
+      <Filter>Header Files\Yoda\container\matrix</Filter>
+    </ClInclude>
+  </ItemGroup>
+</Project>
```

Comment:
No. I should probably add that and set up a build target. This just allows full intellisense and macro expansion in the IDE, which makes it a lot easier to work through VC++ compile errors.